test/clj/cider/nrepl/middleware/inspect_test.clj

```diff
@@ -273,6 +290,109 @@
       (is (re-find #"Page size: 5, showing page: 2 of 20"
                    (extract-text small-page-2))))))
 
+(deftest max-atom-length-integration-test
+  ;; Default max length is 150, so 150 - 3 = 147 chars is too long because we
+  ;; need to leave room for an opening quote and trailing ellipsis "xxxxxxxx...
+  ;; NOTE: We'd prefer to use `orchard.inspect/*max-atom-length*` vs.
+  ;; hard-coding `max-len` to 150 in case the default ever changes, but test
+  ;; code isn't processed by mranderson and so can't refer to inlined deps
+  ;; including orchard.
+  ;; See also https://github.com/benedekfazekas/mranderson/issues/5
+  (let [max-len 150 #_(var-get #'orchard.inspect/*max-atom-length*)
```

Comment:
One other new change is an attempt to avoid hard-coding orchard's defaults here, but I couldn't get mranderson to munge the test code requires. I gave up after a few stabs at editing project.clj `with-profile +test`/`:source-paths`/`:test-paths` entries, and reverted to hard-coding. I think https://github.com/benedekfazekas/mranderson/issues/5 is related.